modules/tinymce/CHANGELOG.md

```diff
@@ -6,66 +6,66 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
-## Added
-- New `sidebar_show` option to show the specified sidebar on initialization #TINY-8710
-- New `newline_behavior` option to control what happens when the enter key is pressed or the `mceInsertNewLine` command is used #TINY-8458
-- New `transparent` property for `iframe` dialog component #TINY-8534
-- New `removeAttributeFilter` and `removeNodeFilter` functions to the `DomParser` and DOM `Serializer` APIs #TINY-7847
-- New `iframe_template_callback` option in the `media` plugin. Patch provided by Namstel #TINY-8684
-- New `dispatchChange` UndoManager API to fire the `change` with current editor status as level and current undoManager layer as lastLevel #TINY-8641
+### Added
+- New `sidebar_show` option to show the specified sidebar on initialization. #TINY-8710
+- New `newline_behavior` option controls what happens when the Return or Enter key is pressed or the `mceInsertNewLine` command is used. #TINY-8458
+- New `transparent` property for `iframe` dialog component. #TINY-8534
+- New `removeAttributeFilter` and `removeNodeFilter` functions added to the DomParser and DOM Serializer APIs. #TINY-7847
+- New `iframe_template_callback` option in the media plugin. Patch provided by Namstel. #TINY-8684
+- New dispatchChange UndoManager API to fire the change with current editor status as level and current undoManager layer as lastLevel. #TINY-8641
 
 ### Improved
-- Clearer focus states for buttons while navigating with keyboard #TINY-8557
-- Support annotating certain block elements directly when using the editor annotator API #TINY-8698
-- The `mceLink` command can now take the value `{ dialog: true }` to always open the link dialog #TINY-8057
-- All help dialog links to `https://www.tiny.cloud` now include `rel="noopener"` to avoid potential security issues #TINY-8834
+- Clearer focus states for buttons while navigating with keyboard. #TINY-8557
+- Support annotating certain block elements directly when using the editor annotator API. #TINY-8698
+- The `mceLink` command can now take the value `{ dialog: true }` to always open the link dialog. #TINY-8057
+- All help dialog links to `https://www.tiny.cloud` now include `rel="noopener"` to avoid potential security issues. #TINY-8834
```

Comment:
```suggestion
- Clearer focus states for buttons while navigating with keyboard. #TINY-8557
- Support annotating certain block elements directly when using the editor's Annotator API. #TINY-8698
- The `mceLink` command can now take the value `{ dialog: true }` to always open the link dialog. #TINY-8057
- All help dialog links to `https://www.tiny.cloud` now include `rel="noopener"` to avoid potential security issues. #TINY-8834
```